nixos/tests/web-apps/kanboard.nix

```
@@ -0,0 +1,25 @@
+import ../make-test-python.nix (
+  { pkgs, ... }:
+
+  {
+    name = "kanboard";
+    meta.maintainers = with pkgs.lib.maintainers; [ yzx9 ];
```

```suggestion
  { lib, ... }:

  {
    name = "kanboard";
    meta.maintainers = with lib.maintainers; [ yzx9 ];
```
`lib` should also be an argument here IIRC